docker-compose.yml

```diff
@@ -26,6 +26,7 @@ services:
     image: itkdev/php8.1-fpm:latest
     networks:
       - app
+      - frontend
```

Comment:
You can add this network if need in override as well

Reply:
Good catch

Reply:
Changed